.php_cs.dist

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Ok :) Too many details, what I needed is if `php-cs-fixer` can look into some `.php_cs/` folder also.